deployment/ansible/deploy.yml

```diff
@@ -117,3 +145,18 @@
     - name: Start epoch daemon
       command: "{{ project_root }}/bin/epoch start"
       tags: [daemon]
+
+    - name: Wait epoch node API to boot
+      wait_for:
+        port: "{{ config.http.external.port }}"
+        host: "{{ ansible_ssh_host }}"
+        timeout: 5
+      connection: local
+      tags: [health-check]
+
+    - name: API health check
+      uri:
+        url: http://{{ ansible_ssh_host }}:{{ config.http.external.port }}/v1/top
+        timeout: 5
```

Comment:
I see this is the timeout for one attempt to connect to the server. I see no retries so I assume 1 (i.e. only one attempt). I recommend:
* Specify retries (e.g. 10);
* Specify time to wait among retries (e.g. 10s);
* Test on a box comparable to UAT with a copy of the DB of current testnet (just in case timing does not work on testnet deployment...).

----

On the code side, I see that [aehttp - the app responsible for starting HTTP listeners - waits on aecore](https://github.com/aeternity/epoch/blob/10a3bf23fe0c155da0ab7152b320c8e7244efced/apps/aehttp/src/aehttp.app.src#L15), and that aecore initialization requires the persisted chain to be read.

Reply:
https://github.com/aeternity/epoch/pull/560/files#diff-2ad1947fe9b869ffd23a6a86872e65e0R158 guarantee that HTTP server is running (with timeout 5). It's internally polling so we don't need retries.

This just checks if API response code is 200 (no internal errors).

Reply:
This should be sufficient for now 602c2c9